api/lib_test.go

```diff
@@ -86,7 +86,7 @@ func mockEnv(sender []byte) types.Env {
 }
 
 func binaryAddr(human string) []byte {
-	res := make([]byte, 42)
+	res := make([]byte, 32)
```

Comment:
:facepalm: why did I do that?

Reply:
Because it makes absolute sense to use non-common test data ensuring to test edge case rather than happy path?